index.js

```diff
@@ -62,7 +68,8 @@ Babelify.configure = function (opts) {
   if (opts.presets && opts.presets._) opts.presets = opts.presets._;
 
   return function (filename) {
-    if (!babel.util.canCompile(filename, extensions)) {
+    var extname = path.extname(filename)
+    if (extensions.indexOf(function (ext) {return ext === extname}) > -1) {
```

Comment:
Is this intended to be `.filter`?

Reply:
I believe it should be `extensions.includes(path.extname(filename))`, right?

Reply:
Node 4 is still supported by Babel, so we'll want to stick to features that work there.

Reply:
That's what I was just checking...

Reply:
that why I went with `indexOf`, I don't think there's any node/browser version that don't support that.

Reply:
@noygal except indexOf takes a value, not a function, as argument :/

Reply:
😳 ...
didn't use for a while now